client/src/containers/home/MapFragment.js

```diff
@@ -1,28 +1,81 @@
 import React from 'react';
 import { Tabs } from 'antd';
-import HereMap from '../../components/HereMap';
+import HereMapInteractive from '../../components/HereMapInteractive';
 
 const { TabPane } = Tabs;
 
 const dummyPoints = [
-  [44.4368, 26.1125],
-  [44.4398, 26.1195],
+  {
+    general_id: 75,
+    risk_category: 'U1',
+    lat: 44.34556,
+    lng: 26.206,
+  },
+  {
+    general_id: 74,
+    risk_category: 'U3',
+    lat: 44.43556,
+    lng: 26.106984,
+  },
+  {
+    general_id: 73,
+    risk_category: 'N/A',
+    lat: 44.39556,
+    lng: 26.156,
+  },
+  {
+    general_id: 72,
+    risk_category: 'RS I',
+    lat: 44.42556,
+    lng: 26.176,
+  },
+  {
+    general_id: 71,
+    risk_category: 'RS II',
+    lat: 44.40553,
+    lng: 26.186,
+  },
+  {
+    general_id: 70,
+    risk_category: 'RS III',
+    lat: 44.41339,
+    lng: 26.146,
+  },
 ];
 
 export default () => {
   return (
     <Tabs size="large" animated={false} defaultActiveKey="1" onChange={null}>
       <TabPane tab="Toate clădirile cu risc seismic" key="all">
-        <HereMap points={dummyPoints} />
+        {/* apikey to be set */}
```

Comment:
I'm not sure what this is for. We're setting `apiKey=""` on the second line this so shouldn't we have a variable or something similar here?